app/queries/district_analytics_query.rb

```diff
@@ -43,12 +43,10 @@ def registered_patients_by_period
   def follow_up_patients_by_period
     @follow_up_patients_by_period ||=
       Patient
-        .joins(:blood_pressures)
         .group('blood_pressures.facility_id')
-        .follow_ups(@period, last: @prev_periods)
+        .hypertension_follow_ups_by_period(@period, last: @prev_periods)
```

Comment:
Is a corresponding change required in a cohort analytics query? Even if that query uses BPs to do stuff, we should explicitly add a `with_hypertension` filter on it.

Reply:
Are you talking about something different from what is already covered in https://github.com/simpledotorg/simple-server/pull/940?

Reply:
Nope, that'll do it. Why do we address the district analytics query in this PR instead of #940?

Reply:
That's a good question.

The trouble is that it was originally using what is now the `encounters` version of the query. If we leave it be then various tests will break because of the strong coupling between the methods used in `DistrictAnalytics` and `Patient`. Instead of spending that effort trying to fix something that needs to get reverted anyway, I made the change right here.

The CohortAnalytics don't have this problem because patients are passed in.

Reply:
Ooh neat. Yup, we're gonna put this all out together anyway, so let's do what's smoothest. 👍